docs/source/architecture.rst

```diff
@@ -0,0 +1,34 @@
+Architecture
+============
+
+Cloudformation
+--------------
+
+Cloudformation will create the CI/CD components utilizing:
+
+    * **CodePipeline**
+    * **CodeCommit**
+    * **CodeBuild**
+    * **s3**
```

Comment:
Again nitpicking: We should use S3 instead of s3.

Reply:
fixed! Ty!